src/components/Dashboard/StaffNotation.vue

```diff
@@ -71,6 +87,19 @@
                 <span class="app-blue">Record Conversion</span>
               </v-list-item-title>
             </v-list-item>
+            <template v-if='isFirm'>
```

Comment:
Please follow existing code style.
1. use " for v-if statements (in template)
2. indent nested blocks